BaristaSwift/Common/Common/Store/AccountStore.swift

```diff
@@ -41,6 +42,24 @@ public class AccountStore: Store<Account> {
         return self.account(userId)
     }
     
+    public func getOrCreateMyAccount() -> Promise<Account> {
```

Comment:
Both apps' AppDelegate had code doing basically what this method does. So I moved it here (and modified to use promises).